azure-functions/acbs-function/.eslintrc.js

```diff
@@ -1,25 +1,33 @@
 module.exports = {
-  extends: 'airbnb-base',
+  extends: ['airbnb-base', 'prettier'],
```

Comment:
Adding `'prettier'` here (available from the `eslint-config-prettier` package adding in the root `package.json`) is what resolves any conflicts between eslint and prettier formatting rules.